### PR TITLE
Refactor community section

### PR DESCRIPTION
**Pull request by jpaddison3**
Stop using the meta flag in favor of a community tag. There are two ~orthogonal parts to this PR:
1. Ripping out all ways in which the EA Forum uses the meta flag and personalBlog flag to refer to the community section.
2. Migrating that data to refer to the community tag.

This is a riskier migration than usual, as we'll want to run the user settings migration simultaneously with deploy, so it's hard to test that part thoroughly with production data.

### CODE REVIEW

**Comment by colophonemes** [comment]
> This is a riskier migration than usual, as we'll want to run the user settings migration simultaneously with deploy, so it's hard to test that part thoroughly with production data.

Could you do a `mongodump` and just run with production data? Might be worth it, esp given no transactions/rollback.

File: packages/lesswrong/server/migrations/2020-08-09-metaToCommunity.ts [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+        }
+      }
+    })
+  }
+})
+
+// Once we've deployed, run this migration to mark all community posts as
+// frontpage, and remove the meta flag
+registerMigration({
+  name: 'moveMetaToFrontpage',
```
Comment (colophonemes) [inline packages/lesswrong/server/migrations/2020-08-09-metaToCommunity.ts:132]: Interested why you're doing this as two passes, given you're operating on exactly the same set of posts as in the migration above?

**Review by colophonemes — COMMENTED** [review]
Haven't tested yet, but here are some comments for you

**Comment by colophonemes** [comment]
The ordering here seems wrong (i.e. it seems random, rather than the most-relevant posts):

image

Ordering on the tag page itself seems fine

image

**Comment by colophonemes** [comment]
Probably not one for this PR, but the `Load More` button on the tag page is _veeeeeery_ slow:

Screen Recording 2020-08-25 at 1 11 43 pm

**Comment by jpaddison3** [comment]
Sweet jesus

**Comment by colophonemes** [comment]
Oh look, the relevance scoring works! Unfortunately it applies to the single post that I've promoted to community, which puts it way down below a bunch of stuff that's below `0` karma.

image

**Comment by jpaddison3** [comment]
I believe lw would say it's working as intended.

**Review by colophonemes — APPROVED** [review]
ASTLAC, nice work

File: packages/lesswrong/lib/publicSettings.ts [hunk 2]
```diff
@@ -73,3 +74,11 @@ export const forumAllPostsNumDaysSetting = new DatabasePublicSetting<number>('fo
 
 export const localeSetting = new DatabasePublicSetting<string>('locale', 'en-US')
 export const legacyRouteAcronymSetting = new DatabasePublicSetting<string>('legacyRouteAcronym', 'lw') // Because the EA Forum was identical except for the change from /lw/ to /ea/
+
+// frontpageFitlerSettings default tag filter
```
[the remark this answers is not in the document]
Comment (jpaddison3) [reply packages/lesswrong/lib/publicSettings.ts:78]: I had this in a variable somewhere before. My eyes really edit that out.

**Comment by jpaddison3** [comment]
> The ordering here seems wrong

This will be fixed as soon as I deploy the merge, which should be very soon. And it's not a bad bug, so I'm inclined to leave it.

**Pull request merged**